smsgateway/backends/jasmin.py

```diff
@@ -1,8 +1,10 @@
+# -*- coding: utf-8 -*-
+
 from datetime import datetime
 
-from urllib2 import urlopen
+from six.moves.urllib.request import urlopen
+from six.moves.urllib.parse import urlencode
```

Comment:
By using this version of urlencode instead of the django one we avoid encoding the message in utf-8. This way we can properly encode in latin-1 as the jasmin backend is expecting. This should prevent messed up characters in french sms messages. Has this every worked properly when using Jasmin or have we been sending wrongly encoded messages for months already ?